azurerm_dns_zone/outputs.tf

```
@@ -0,0 +1,14 @@
+output "dns_zone_name" {
```

```suggestion
output "name" {
```

Maybe just name? I don't know if there is a naming convention here, but I saw in here https://github.com/pagopa/io-infrastructure-modules-new/blob/master/azurerm_subnet/outputs.tf if was just name.